docs/api.md

```diff
@@ -28,13 +28,27 @@ Distinct API versions present more clear, consistent views of system resources a
 
 The [API and release versioning proposal](versioning.md) describes the current thinking on the API version evolution process.
 
-## v1beta1 and v1beta2 are deprecated; please move to v1beta3 ASAP
+## v1beta1, v1beta2, and v1beta3 are deprecated; please move to v1 ASAP
 
-As of April 1, 2015, the Kubernetes v1beta3 API has been enabled by default, and the v1beta1 and v1beta2 APIs are deprecated. v1beta3 should be considered the v1 release-candidate API, and the v1 API is expected to be substantially similar. As "pre-release" APIs, v1beta1, v1beta2, and v1beta3 will be eliminated once the v1 API is available, by the end of June 2015. 
+As of June 4, 2015, the Kubernetes v1 API has been enabled by default. The v1beta1 and v1beta2 APIs were deleted on June 1, 2015. v1beta3 is planned to be deleted before the end of June.
 
-## v1beta3 conversion tips
+### v1 conversion tips (from v1beta3)
 
-We're working to convert all documentation and examples to v1beta3. Most examples already contain a v1beta3 subdirectory with the API objects translated to v1beta3. A simple [API conversion tool](cluster_management.md#switching-your-config-files-to-a-new-api-version) has been written to simplify the translation process. Use `kubectl create --validate` in order to validate your json or yaml against our Swagger spec. 
+We're working to convert all documentation and examples to v1. A simple [API conversion tool](cluster_management.md#switching-your-config-files-to-a-new-api-version) has been written to simplify the translation process. Use `kubectl create --validate` in order to validate your json or yaml against our Swagger spec.
+
+Some important difference between v1beta3 and v1:
+
+* Container's `privileged` or `capabilities` properties were nested under the `securityContext` property. See [Security Contexts](security_context.md).
+* Container State's `termination` property was renamed to `terminated`.
+* PodSpec's `host` property was renamed to `nodeName`.
+* PodStatus's `Condition` property was renamed to `conditions`.
+* ServiceSpec's `portalIP` property was renamed to `clusterIP`.
+* ServiceSpec's `createExternalLoadBalancer` property was removed. This is infered to be true when `type` is `LoadBalancer`.
+* ServiceSpec's `publicIPs` property was deprecated and now called `deprecatedPublicIPs`. This property will be removed entirely when the default storage format in moved from v1beta3 to v1. Users of this should switch to `NodePort` service types. Read [External Services](services.md#external-services) for more info.
```

Comment:
I would add a bit more here.  PublicIPs was hard to get people to understand, and now that people are using it, we are taking it away.

ServiceSpec's `publicIPs` property was deprecated and now called `deprecatedPublicIPs`. This property will be removed entirely when v1beta3 is removed. The vast majority of users of this field were using it to expose Services on ports on the Node.  Those users should switch to `NodePort` service types. Read [External Services](services.md#external-services) for more info.  If this is not sufficient for your use case, please contact @thockin or file an issue.